fileglancer/handlers.py

```diff
@@ -93,15 +111,21 @@ def _get_filestore(self, path_name):
         return filestore
 
 
-    @web.authenticated
```

Comment:
This is temporary, until we can get a file proxy in the central server.

Reply:
What was the auth preventing? Without this present, anyone can request that endpoint and get a response.